BanIpNotificationEmail.php

```diff
@@ -36,10 +36,15 @@ public function send($ipRange, $ip, $email, $maxActionsAllowed, $locationData, $
             $mail->setFrom($mail->getFrom(), 'Web Analytics Reports');
         }
 
+
         $mailBody = 'This is for your information. The following IP was banned because visit tried to track more than ' . Common::sanitizeInputValue($maxActionsAllowed) . ' actions:';
         $mailBody .= PHP_EOL . PHP_EOL . '"' . Common::sanitizeInputValue($ipRange) . '"' . PHP_EOL;
         $instanceId = SettingsPiwik::getPiwikInstanceId();
-
+        $matomoUrl = SettingsPiwik::getPiwikUrl();
+        if (!empty($matomoUrl)) {
+            $url = parse_url($matomoUrl);
+            $matomoHost = $url['host'];
```

Comment:
Maybe this would be a little cleaner?
```suggestion
            $matomoHost = parse_url($matomoUrl, PHP_URL_HOST);
```